ibis/common/patterns.py

```diff
@@ -688,39 +693,6 @@ def __getattr__(self, name: str) -> Pattern:
         return self.pattern(getattr(self.module, name))
 
 
-class Apply(Slotted, Pattern):
```

Comment:
Consider to keep it as a specialized pattern for `CoercedTo(type not implementing Coercible)`